x/evm/types/params.go

```diff
@@ -46,7 +46,7 @@ func DefaultParams() Params {
 		EvmDenom:     DefaultEVMDenom,
 		EnableCreate: true,
 		EnableCall:   true,
-		ExtraEIPs:    []int64{2929, 2200, 1884, 1344},
+		ExtraEIPs:    []int64{},
```

Comment:
add a comment on DefaultParams that ExtraEIPs is empty to prevent overriding the latest hard fork instruction set

Reply:
done